extensions/amp-story/1.0/amp-story.js

```diff
@@ -1003,7 +1009,7 @@ export class AmpStory extends AMP.BaseElement {
         // Preloads and prerenders the share menu.
         this.shareMenu_.build();
 
-        const infoDialog = shouldShowStoryUrlInfo(this.viewer_)
+        const infoDialog = this.viewer_ && shouldShowStoryUrlInfo(this.viewer_)
```

Comment:
The viewer should always be defined at this point. This is dangerous because it means that `infoDialog` would be false if the viewer is not defined, without any error message. If you want to satisfy the compiler, please use `devAssert()`.

Reply:
`devAssert()` is making sense now. Thanks Gabriel!